.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
- [ ] You did a great job 😃 at the linter, but I see that the linter is not appearing in the checks, so please try to update the linters.yml file with the new one to add the linter of js maybe the problem is there just, please try to follow these steps in this [Page](https://github.com/microverseinc/linters-config/tree/master/javascript) to set up the js lint correctly 💯